cli/azd/pkg/infra/provisioning/provisioning_progress_display.go

```diff
@@ -23,31 +23,29 @@ const succeededProvisioningState string = "Succeeded"
 type ProvisioningProgressDisplay struct {
 	// Keeps track of created resources
 	createdResources map[string]bool
-	subscriptionId   string
-	deploymentName   string
 	resourceManager  infra.ResourceManager
 	console          input.Console
+	scope            infra.Scope
 }
 
-func NewProvisioningProgressDisplay(rm infra.ResourceManager, console input.Console, subscriptionId string, deploymentName string) ProvisioningProgressDisplay {
+func NewProvisioningProgressDisplay(rm infra.ResourceManager, console input.Console, scope infra.Scope) ProvisioningProgressDisplay {
 	return ProvisioningProgressDisplay{
 		createdResources: map[string]bool{},
-		subscriptionId:   subscriptionId,
-		deploymentName:   deploymentName,
+		scope:            scope,
 		resourceManager:  rm,
 		console:          console,
 	}
 }
 
 // ReportProgress reports the current deployment progress, setting the currently executing operation title and logging progress.
-func (display *ProvisioningProgressDisplay) ReportProgress(ctx context.Context) (*DeployProgress, error) {
+func (display *ProvisioningProgressDisplay) ReportProgress(ctx context.Context, scope infra.Scope) (*DeployProgress, error) {
```

Comment:
Does `scope` still need to be passed to `ReportProgress` if it's also part stored in the struct? I think the lifetime should be with the object (at least the way it was written initially)